opentech/apply/funds/models.py

```diff
@@ -251,6 +252,7 @@ class Round(WorkflowStreamForm, SubmittableStreamForm):  # type: ignore
     parent_page_types = ['funds.FundType']
     subpage_types = []  # type: ignore
 
+    lead = models.ForeignKey(settings.AUTH_USER_MODEL, limit_choices_to={'is_staff': True})
```

Comment:
The round should inherit the lead from the Fund type. Is that covered somewhere else (i.e. another user story)?

Reply:
Yes, there is another story for this.